frontend/tests/e2e/features/addStudent.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Add new students
+    As a user
+    I want to add new students
+    So that I can view or manipulate their details later
+
+Background:
+    Given the user has already logged in and navigated to the home page with username "admin" and password "admin"
+
+Scenario Outline: Add a new valid student with no ielts
+    When the user tries to add a student with name "<name>", email "<email>", address "<address>", phone "<phone>", qualification "<qualification>", cgpa "<cgpa>", destination "<destination>" and ielts "<ielts>"
+    Then the user should be provided with add successful message
+    Examples:
```

Comment:
Better to use dataTable instead of an example table. A single example is sufficient for this test.